src/main/java/Area.java

```diff
@@ -0,0 +1,3 @@
+public interface Area {
+    void drawFigure();
```

Comment:
Why drawFigure? Simple draw() - it is importent!
 What if tomorrow you will create class Plane implements Drawable ?